test/api/unit/serializers/site.test.js

```diff
@@ -64,6 +65,19 @@ describe('SiteSerializer', () => {
       expect(object.basicAuth.password).to.eq('**********');
       expect(object.organizationId).to.equal(org.id);
     });
+
+    it('includes URL editability when associated to domain', async () => {
+      const site = await factory.site({ domain: 'https://www.agency.gov', demoDomain: null });
+      const domain = await factory.domain.create({ siteId: site.id, names: 'www.agency.gov', context: "site" });
```

Comment:
```suggestion
      const domain = await factory.domain.create({ siteId: site.id, names: 'www.agency.gov', context: 'site' });
```

Reply:
Whoops. Weird that eslint didn't complain about that.